portfolio/src/main/webapp/index.html

```diff
@@ -2,18 +2,63 @@
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click <a href="/hello">here</a> to see content created by server code.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
-    </div>
+      <h1>My Portfolio: Manuela Rodriguez</h1>
+      <p>My name is Manuela Rodriguez, I'm a third-year student at New York University studying Computer Science and Business 
+          with a concentration in Finance. I am passionate about using data and technology to solve complex and ambiguous problems,
+          and making Computer Science more accessible to historically underrepresented communities.
+      </p>
+    
+      <button onclick="showGreetingMessage()"> Get a string!</button>
+      <div id="intro-container"> </div>
+
+      <p> <img src="/images/headshot.JPG" width="467.8" height="311.9"/> </p>
+      <p>Click here to get a random fact about me:</p>
+      <button onclick="addRandomFact()">Get a fact!</button>
+      <div id="fact-container"></div>
+
+      <h3>Check out my LinkedIn and Github profiles!</h3>
+      <ul> 
+      <li><a href="https://www.linkedin.com/in/manuela-rodriguez-a8a3b4157/">LinkedIn</a></li>
+      <li><a href="https://github.com/manuelarod">Github</a></li>
+      </ul>
+
+      <h2> Projects </h2>
+      <h4> National Museum of American Diplomacy ImpactHack 2020</h4> 
+      <ul>
+          <li> Worked in a team of four to build a full-stack web application where users can take an 
+                interactive quiz that matches their skills and interests to careers in diplomacy. </li>
+                <li> Designed an efficient data structure and ranking algorithm to match the user’s responses 
+                  with career tracks in diplomacy.</li>
+            <li><a href="https://hidden-shore-16633.herokuapp.com/">See project</a></li>
+      </ul>
+      <h4> The Restaurant Industry Amidst the COVID-19 Pandemic</h4>  
+      <ul>
+                <li> Used publicly available data to provide stakeholders 
+                with an insightful view of the state of the restaurant industry in the U.S.</li>
+                <li> Performed data cleaning, plotted graphs using Pandas, Matplotlib, and Seaborn, and created metrics such as the week-over-week percentage change 
+                    in seated diners to analyze the industry’s response as markets reopen.</li>
+                    <li><a href="https://github.com/manuelarod/restaurant-industry-amidst-covid19">See project</a></li>
+        </ul>
+            <h4> Hydropower Generators in the U.S.</h4>
+            <ul><li> Used U.S. hydropower generator data to build decision tree models and random forests, with an accuracy of 93% and 95%
+respectively, to predict if a generator was retired and determine whether the Hydro infrastructure in the U.S. is worth renovating. </li>
```

Comment:
Not sure if it's just my phone displaying this differently, but make sure to run this through a code formatted real quick!